composer/workflows/requirements.txt

```diff
@@ -6,3 +6,4 @@ apache-airflow-providers-cncf-kubernetes==4.1.0
 apache-airflow-providers-google==8.0.0
 scipy==1.8.1; python_version > '3.0'
 apache-airflow-providers-amazon==4.0.0
+apache-airflow-providers-microsoft-azure==3.7.2
```

Comment:
Any reason we can't use 4.0.0? (I only ask because I used it yesterday)

Reply:
I used it because this version is on https://raw.githubusercontent.com/apache/airflow/constraints-2.2.5/constraints-3.8.txt. I can change it to if there's a preference to use the latest version.

Reply:
that reasoning is fine!